Missing pytz dependency breaks calendar tools import

## Description

The recent calendar changes (out-of-office support) import `pytz` in `gcalendar/calendar_tools.py`, but `pytz` is not listed in `pyproject.toml` dependencies. This causes a silent import failure at startup:

```
[ERROR] Failed to import tool 'calendar': No module named 'pytz'
```

Calendar tools are silently dropped from the server — all other services load fine.

## Steps to reproduce

```bash
uv sync --all-extras
uv run main.py --tools calendar
# Calendar tools fail to register
```

## Expected behavior

Calendar tools load successfully, or `pytz` is added to `pyproject.toml`.

## Environment

- Version: 1.17.0
- Python: 3.11
- OS: Windows 11

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -23,6 +23,7 @@ dependencies = [
  "pyyaml>=6.0.2",
  "cryptography>=45.0.0",
  "defusedxml>=0.7.1",
+ "pytz>=2026.1.post1",
 ]
 classifiers = [
     "Development Status :: 4 - Beta",

